source/extensions/filters/http/lua/BUILD

```diff
@@ -21,8 +21,8 @@ envoy_cc_library(
         "//source/common/common:enum_to_int",
         "//source/common/config:well_known_names",
         "//source/common/http:message_lib",
-        "//source/common/lua:lua_lib",
-        "//source/common/lua:wrappers_lib",
+        "//source/extensions/filters/common/lua:lua_lib",
+        "//source/extensions/filters/common/lua:wrappers_lib",
```

Comment:
build comment?  
Follow up fine, or we can just audit them all at the end.

Reply:
Sorry missed this. I will fix in the dynamo change since I have to merge master there anyway.